libcperciva/datastruct/elasticarray.c

```diff
@@ -28,6 +28,12 @@ resize(struct elasticarray * EA, size_t nsize)
 	if (EA->alloc < nsize) {
 		/* We need to enlarge the buffer. */
 		nalloc = EA->alloc * 2;
+		/*
```

Comment:
Sorry, one more nitpick: please add a blank line before this comment.

Reply:
Actually, on second thought, never mind.

I don't know why github isn't running the Actions tests on this PR.  Based on the documentation I've found, it should be working now.  :(

I'm going to make a new PR (from my own account) with these commits, plus that "blank line" nitpick.  That's the only guaranteed way I know of getting github Actions to work, and that'll let the review go through faster.